vuu-ui/packages/vuu-layout/src/layout-persistence/RemoteLayoutPersistenceManager.ts

```diff
@@ -134,13 +138,49 @@ export class RemoteLayoutPersistenceManager
   }
 
   saveApplicationLayout(layout: LayoutJSON): Promise<void> {
-    // TODO POST api/layouts/application #71
-    console.log(layout);
-    return new Promise((resolve) => resolve());
+    return new Promise((resolve, reject) =>
+      fetch(`${baseURL}/${applicationLayoutsSaveLocation}`, {
+        method: "PUT",
+        headers: {
+          "Content-Type": "application/json",
+          "username": "vuu-user"
+        },
+        body: JSON.stringify(layout),
+      })
+      .then((response) => {
+        if (!response.ok) {
+          reject(new Error(response.statusText));
+        }
+        resolve();
+      })
+      .catch((error: Error) => {
+        reject(error);
+      })
+    );
   }
 
   loadApplicationLayout(): Promise<LayoutJSON> {
-    // TODO GET api/layouts/application #71
-    return new Promise((resolve) => resolve(defaultLayout));
+    return new Promise((resolve, reject) =>
+      fetch(`${baseURL}/${applicationLayoutsSaveLocation}`, {
+        method: "GET",
+        headers: {
+          "username": "vuu-user",
+        },
+      })
+      .then((response) => {
+        if (!response.ok) {
+          reject(new Error(response.statusText));
+        }
+        response.json().then((response: ApplicationLayout) => {
+          if (!response) {
+            reject(new Error("Response did not contain valid application layout information"));
+          }
+          resolve(response.definition);
```

Comment:
Since we're only ever using the definition, it's probably worth changing the data contract to remove the username from the response. I've raised a [ticket](#102) for this to avoid conflicts with ongoing work (and scope creep).